e2e/tests/popout.spec.ts

```diff
@@ -22,7 +22,7 @@ test.describe('popout window', () => {
         const text = await popOut.textContent('#calls-popout-mute-button');
         expect(text).toBe('Unmute');
 
-        await devPage.leaveCall();
+        await popOut.locator('.button-leave').click();
```

Comment:
Interesting. This should work regardless of whether the user leaves from the pop-out window or the widget.